discovery-provider/src/queries/get_skipped_transactions.py

```diff
@@ -1,12 +1,17 @@
 import logging
 
 import redis
+from sqlalchemy import func
 from src.models.indexing.block import Block
 from src.models.indexing.skipped_transaction import SkippedTransaction
 from src.utils import db_session, helpers
 from src.utils.config import shared_config
 from src.utils.redis_cache import get_json_cached_key, set_json_cached_key
 
+# The maximum number of skipped transactions allowed
+MAX_SKIPPED_TX = 10
```

Comment:
confirming is this supposed to be 5 or 10? i saw to alec's comment you said 5 but this looks like 10 unless i'm misreading the code

Reply:
oh yea it was 5 but i made it 10